app/src/main/java/com/example/moviedetailsapplication/network/RetrofitApi.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.moviedetailsapplication.network
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitApi {
+
+    var BASE_URL = "https://api.themoviedb.org/3/"
+
+    fun getClient(): Retrofit {
+
+        val retrofit = Retrofit.Builder()
+            .addConverterFactory(GsonConverterFactory.create())
+            .baseUrl(BASE_URL)
+            .build()
+        return retrofit
```

Comment:
Small thing, it will create new retrofit client every time we call getClient(). instead we can make a single instance and use it